core/blockchain.go

```diff
@@ -258,6 +258,9 @@ type BlockChain struct {
 	forker     *ForkChoice
 	vmConfig   vm.Config
 	logger     *tracing.Hooks
+
+	// note: added to assist debugging in case of a failed validation after bundler performed second validation
+	rip7560TransactionDebugInfos []*types.Rip7560TransactionDebugInfo
```

Comment:
VERY hacky save-state-i-want-to-use-it-later.

Reply:
I actually suggested a differnt API: a back-api from node to bundler "sendFailedTransactions"
I originally thought it should only contain txhash, but adding debug info to it is nice addition.

Reply:
I still think that the node should push the failed  TXs to the bundler.
that is the flow is:

- fillTransactions(): 
   - PendingRip7560Bundle()
      - internally calls:  node -> bundler: aa_getRip7560Bundle (from getPending
   - commitRip7560TransactionsBundle
   + add:
         bundler->node: aa_failedRip7560()


(also: rename PendingRip7560Bundle, since it is not "pending" queue - it is actively contacting bundler to get list of candidates.

Reply:
It can simplify the state in geth if it can simply push failed txs back to the bundler, and let the bundler handle the implications of that for the user. It makes sense for the separation we made in the user-bundler-node model.